public/swagger/Site.json

```diff
@@ -67,6 +68,17 @@
     "previewLink": {
       "type": "string"
     },
+    "basicAuth": {
```

Comment:
should this just be a reference to the BasicAuth.json schema?

Reply:
i don't think its necessary to maintain a separate file for basicAuth. initially BasicAuth.json was its own file because it was an api response all on its own.  now its part of the site response